code/game/objects/items/devices/scanners.dm

```diff
@@ -127,6 +127,10 @@ GENE SCANNER
 					 \n<span class='info'>Body temperature: ???</span>")
 		return
 
+	if(ispodperson(M)&& !advanced)

```

Comment:
does this affect those pay-per-usse health scanners that no one ever uses?

Reply:
I'll check ! I don't this particulat method is used outside of health analyzers and PDA though.

Reply:
It works as intended.